### PR TITLE
Updates the caravan space ruin.

### PR DESCRIPTION
**Pull request by WJohn**
~~This re-adds the 2nd fighter ship as was originally seen in #34621 but was unintentionally removed in #37474~~ Removed at ninjanomnom's request, as having a bandaid fix will put-off incentive to fix this more properly.

This also nerfs a lot of this ruin's loot. The middle cargo ship containing all the guns has lost much of its firepower; the 40mm shells were moved into the pirate ship and dropped to only one box, the frag12 shells are gone, the shotgun shells crate has been dropped to a single rubber shells crate, and the 10mm boxes to two magazines. The APS pistol has been moved into the cargo ship, while one of the energy pistols moved outside. The ship also now contains a bandolier and a shoulder holster. Also, the two 8 capacity riot shotguns have been dropped to a single 4 capacity shotgun. Tech shells crate is mostly unchanged, removing a frag 12 and the empty tech shells, and instead adding 2 stun shells.

The pirate ship has lost its combat shotgun and one of its laser guns, as stated before it now contains the 40mm shells box.

The dropship now has a pistol, among a bunch of clothing, but no spare 10mm magazines in its lockers.

Overall should make the ruin much less easy to cheese, and change up the meta a bit now that you're not drowning in lethal buckshot rounds, and there's an extra ship hanging around waiting to shoot you.

:cl: WJohn
balance: Caravan ruin overall contains much less firepower, and should be harder to cheese due to placement changes and less ammunition to work with.
/:cl:

### CODE REVIEW

**Comment by subject217** [comment]
Remove 👏 the 👏 syndie 👏 headsets 👏 

To clarify, they spawn in the lockers of the Syndicate Dropship. Two headsets, both with Syndicate encryption keys. I wasn't able to find any other headset that spawned with a Syndie key in the ruin, including on all the mobs.

**Comment by WJohn** [comment]
Oh! So they are.

**Comment by ninjanomnom** [comment]
As I said in irc if you want to include the second fighter it needs to be different from the first or I need to get around to moving some code from pods to regular shuttles to allow duplicate templates. I can't tell if there are any differences in this fighter compared to the other.

**Comment by WJohn** [comment]
It is different. Different area, mobile port, consoles, everything. It just looks identical to the first because it's the same "ship model". You'd expect these to be mass produced.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -446,8 +446,8 @@
 	suffix = "syndicate_dropship"
 	name = "Syndicate Dropship"
 
-/datum/map_template/shuttle/ruin/syndicate_fighter
-	suffix = "syndicate_fighter"
+/datum/map_template/shuttle/ruin/syndicate_fighter_1
+	suffix = "syndicate_fighter_1"
```
Comment (ninjanomnom) [inline code/datums/shuttles.dm]: you left this change in

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**